src/deluge/gui/colour/rgb.h

```diff
@@ -178,6 +178,17 @@ class RGB {
 		});
 	}
 
+	/**
+	 * @brief Compare two colours to determine if they're the same
+	 *
+	 * @param sourceA The first colour
+	 * @param sourceB The second colour
+	 * @return bool The true / false result if they're the equal
+	 */
+	[[nodiscard]] static constexpr bool isEqual(RGB sourceA, RGB sourceB) {
+		return ((sourceA.r == sourceB.r) && (sourceA.g == sourceB.g) && (sourceA.b == sourceB.b));
+	}
+
```

Comment:
should implement `operator==` instead (and in this cause, you can probably do that using:
```suggestion
	bool operator==(RGB const &) = default;
```

Reply:
gives me an error:

/Users/sean/Documents/GitHub/DelugeFirmware/src/deluge/gui/colour/rgb.h:185:14: error: defaulted 'bool RGB::operator==(const RGB&)' must be 'const'
  185 |         bool operator==(RGB const &) = default;

Reply:
oops, need a `const` before the `=`

Reply:
Awesome worked thanks!